tox.ini

```diff
@@ -10,7 +10,12 @@ commands =
     coverage combine
     coverage report
     coverage xml
-passenv = DATABASE_URL PORT aws_access_key_id aws_secret_access_key
+passenv = 
+    DATABASE_URL PORT 
+    aws_access_key_id 
+    aws_secret_access_key 
+    is_aws_mturk_requestor 
```

Comment:
I like the idea of expecting the configured AWS account to be a requester. I think there should be a way to enable/disable running tests which actually write to AWS/MTurk, but there's probably a more elegant way to do this. A bit more thought required.